src/TipTapEditor/CollapsibleBlock/CollapsibleBlock.css

```diff
@@ -1,16 +1,16 @@
 .collapsible-block {
   /* @apply debug; */
-  @apply flex my-2 p-2 pl-0 rounded-lg;
+  @apply my-2 flex rounded-lg p-2 pl-0;
```

Comment:
There's some diff noise here from running `yarn fmt`

Reply:
Somehow the auto-sort for did not pick up css files.